lib/format.ex

```diff
@@ -15,7 +15,12 @@ defmodule Elchemy.Format do
   def inspect(term) when is_tuple(term) do
       [head | _rest] = term |> Tuple.to_list
       if is_atom(head) do
-          term |> Tuple.to_list() |> Enum.map(&inspect/1) |> Enum.join(" ") |> String.capitalize
+        inner = term |> Tuple.to_list() |> Enum.map(&inspect/1) |> Enum.join(" ")
+        if(String.length() == 1) do ###JAK TO ZROBIC?
+          term |> String.upcase()
```

Comment:
Why have this branch at all, why not just capitalize regardless as if it is a single character then it should(?) get capitalized anyway?

Reply:
with String.capitalize()
```
left:  "A 1 b 2 3 4"
right: "A 1 (B 2 3) 4"
```
with String.upcase()
```
left:  "A 1 b 2 3 4"
right: "A 1 (B 2 3) 4"
```

Reply:
That seems to be precisely what I expect from them though?

Also it doesn't matter as the first branch is only taken when the string has a length of 1 (assuming `term` is supposed to be in it), so it wouldn't matter anyway?